org.upbge.UPBGE.yml

```diff
@@ -0,0 +1,137 @@
+id: org.upbge.UPBGE
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: blender
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--share=network"
+  - "--filesystem=host"
+  - "--filesystem=/run/spnav.sock:ro"
+  - "--env=SPNAV_SOCKET=/run/spnav.sock"
+  - "--env=TMP_DIR=/tmp"
+  - "--env=TMP=/tmp"
+add-extensions:
+  org.upbge.UPBGE.Codecs:
+    directory: lib/codecs
+    add-ld-path: lib
+    bundle: true
+    autodelete: true
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: "."
+cleanup-commands:
+  - mkdir -p /app/lib/ffmpeg
+modules:
+  - shared-modules/libdecor/libdecor-0.1.1.json
+  - name: x264
+    config-opts:
+      - "--prefix=/app/lib/codecs"
+      - "--enable-lto"
+      - "--enable-pic"
+      - "--enable-shared"
+      - "--disable-cli"
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        commit: baee400fa9ced6f5481a728138fed6e867b0ff7f
+    cleanup:
+      - "/lib/codecs/include"
+      - "/lib/codecs/lib/pkgconfig"
+  - name: ffmpeg
```

Comment:
Why do you need ffmpeg separately? You added the extension. If you need to make the extension available at build time, there's an option for that.

Reply:
No idea, I removed it and the FFMPEG features work fine. Please take it [here](https://github.com/flathub/org.blender.Blender) if you're wondering, since I'm just aiming for consistency.